### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by dieum512**
In this branch I:

- Updated the index.html file to match with the desktop version
- Styled the desktop version with media queries
- Checked for some linters errors
- Corrected linters errors
- updated the header of the desktop version
- Styled the updated head
- Made some background images contain the page

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -412,3 +412,158 @@ textarea {
...
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    color: #172b4d;
+    margin-bottom: 15px;
+  }
+
+  .me {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
```
Comment (binhussen) [inline style.css]: - [optional ] kindly, use meaning full class names to make your code understandable  for others 😘😂

File: style.css [hunk 2]
```diff
@@ -412,3 +412,158 @@ textarea {
...
+    border-radius: 0 0 0 150px;
+  }
+
+  .header {
+    padding: 25px 12%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    color: #6070ff;
+  }
```
Comment (binhussen) [inline style.css:431]: - Kindly, make your navigation responsive as given [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)👌👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

😎️
Kindly, make sure you stick to the Figma design.

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @dieum512 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
- Kindly make sure to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @dieum512, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good UI design :heavy_check_mark: 
- Linter checks passing :+1: 
- Good commit history :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**